### ISSUE TITLE
Feedback 'From: orta therox Scrolling fast on iPad Artsists'

### ISSUE THREAD

**Issue by artsyit** [artsy/eigen#1535]
### Message from Orta Therox sent on 2016-05-24 02:09:17 UTC###

> From: orta therox
> 
> Scrolling fast on iPad Artsists

### Link to HockeyApp ###

* [https://rink.hockeyapp.net/manage/apps/37020/feedback/555175](https://rink.hockeyapp.net/manage/apps/37020/feedback/555175)

**Comment by orta** [comment]
This actually isn't a bug, weirdly it was that the works looks like they were broken, cause that was the work

**Comment by alloy** [comment]
Can you rephrase this? I don’t understand what the original problem was nor what your follow-up explanation is trying to say.
